lib/src/rules/use_full_hex_values_for_flutter_colors.dart

```diff
@@ -54,7 +54,7 @@ class _Visitor extends SimpleAstVisitor {
   @override
   void visitInstanceCreationExpression(InstanceCreationExpression node) {
     if (node.staticElement?.library?.name == 'dart.ui' &&
-        node.staticElement?.returnType?.name == 'Color' &&
+        node.staticElement?.returnType?.element?.name == 'Color' &&
         node.staticElement?.name == '') {
```

Comment:
I'd replaced this code with a call to something more generic, like `isConstructorElement(ConstructorElement element, {@required String uriStr, @required String className, @required String constructorName})`.

It is better to check `ConstructorElement.enclosingElement`, which is `ClassElement` than `returnType.element`. Although we probably should override `ConstructorElement.returnType` to return `InterfaceType`.

Reply:
Good idea(s).  Updated.  Thanks!